x-pack/solutions/observability/plugins/observability/server/services/alert_data.ts

```diff
@@ -6,16 +6,51 @@
  */
 
 import { omit } from 'lodash';
-import { CustomThresholdParams } from '@kbn/response-ops-rule-params/custom_threshold';
 import type { AlertsClient } from '@kbn/rule-registry-plugin/server';
-import { DataViewSpec } from '@kbn/response-ops-rule-params/common';
 import {
   ALERT_RULE_PARAMETERS,
   ALERT_RULE_TYPE_ID,
   ALERT_RULE_UUID,
-  OBSERVABILITY_THRESHOLD_RULE_TYPE_ID,
   fields as TECHNICAL_ALERT_FIELDS,
 } from '@kbn/rule-data-utils';
+import { CustomThresholdParams } from '@kbn/response-ops-rule-params/custom_threshold';
+import { OBSERVABILITY_THRESHOLD_RULE_TYPE_ID } from '@kbn/rule-data-utils';
```

Comment:
@cesco-f `OBSERVABILITY_THRESHOLD_RULE_TYPE_ID` previously was also being imported from `'@kbn/rule-data-utils'`. Why did you separate the import here? Is this because of `as TECHNICAL_ALERT_FIELDS`?